src/seedu/addressbook/data/AddressBook.java

```diff
@@ -68,6 +68,11 @@ public UniquePersonList getAllPersons() {
         return new UniquePersonList(allPersons);
     }
 
+    public UniquePersonList sortedAllPersons() {
+        allPersons.sort();
+        return new UniquePersonList(allPersons);
```

Comment:
Do you really need to return a new UniquePersonList if you convert it to an immutableListView in your SortCommand?